### PR TITLE
Test: Refactor K8sValidatedPolicyTestAcrossNamespaces test.

### PR DESCRIPTION
**Pull request by eloycoto**
Deleted `K8sValidatedPolicyTestAcrossNamespaces` because no value added,
both test can be used in `K8sValidatedPolicyTest`.

Two new tests added in Policies:

1- MatchExpression, to test that a custom expression/label will work
correctly. Was testing using namespace label.
2- CNP on namespace. Test to check K8sT policies accross namespace and
to make sure that works correctly.

This test should be faster; no manifest deploys, already used pods.

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4694)

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

File: test/k8sT/Policies.go [hunk 1]
```diff
@@ -331,6 +323,23 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 
 		}, 500)
 
+		It("CNP test matchExpressions key", func() {
```
Comment (ianvernon) [inline test/k8sT/Policies.go]: nit: "matchExpressions" --> "MatchExpressions"

File: test/k8sT/Policies.go [hunk 2]
```diff
@@ -935,182 +946,36 @@ EOF`, k, v)
...
-		case helpers.Create:
-			kubectl.CreateResource(namespace, ns).ExpectSuccess(
-				"cannot create namespace %s", ns)
 
-		case helpers.Delete:
-			kubectl.DeleteResource(namespace, ns).ExpectSuccess(
-				"cannot delete namespace %s", ns)
-		}
-	}
+		It("Cilium Network policy using namespace label and  L7", func() {
```
Comment (ianvernon) [inline test/k8sT/Policies.go]: Remove the extra space between "and  L7"

**Review by ianvernon — APPROVED** [review]
Nothing blocking, just formatting nits. Great change, I'm all for simplicity and fewer files to manage!

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

**Pull request merged**